apptests/tests-doc.md

```diff
@@ -0,0 +1,50 @@
+# Application Specific Tests
+
+These tests are intended to test the configuration of each app. They are not intended to test the functionality 
+of the application.The tests are written in Ginkgo and Gomega.
+
+Most of the tests run in kind locally. The notable exception are the Rook Ceph tests and the Velero Local backup tests 
+which don't work on ARM (Apple Silicon) running Colima. There are some images that we are not building for ARM right now.
+
+## Prerequisites
+
+- [Ginkgo](https://onsi.github.io/ginkgo/)
+- [Kind](https://kind.sigs.k8s.io/)
+- [Docker](https://www.docker.com/)
+- [Colima]()
+
+Setup the environment:
+
+If you are using `colima`, you can set the `DOCKER_HOST` environment variable to the socket used by `colima`. 
```

Comment:
nit: add a note(no need for the actual command) that Rancher Desktop also need this